---
permalink: /dgt/
title: "RIKEN - CLST (2016)"
excerpt: "Center for the Life Science Technologies - Division of Genomic Technologies - GMT Unit   (6 months internship - Yokohama, Japan)"
layout: single
header:
  overlay_image: /assets/images/header-dgt.jpg
  overlay_filter: 0.7
  cta_label: "Visit the research group website"
  cta_url: "http://www.riken.jp/en/research/labs/clst/genom_tech/life_sci_accel/genom_mini/"
  caption: "Photo credit: [**Population transcriptomics**](http://population-transcriptomics.org/)"
sidebar:
  nav: "cv"
modified: 2018-09-29T19:20:00
---

{% include base_path %}

For my Masters' thesis I built an interactive [tool called "Vizection"](https://github.com/shamansim/Vizection) to qualitatively
			and quantitatively analyse single-cell transcriptomic datasets. Each step
			is expected to answer a specific point as for example identify the
			outliers, define sub-populations or identify the genes contributing the
			most to these sub-populations. I developped this tool in close collaboration
			with the molecular biologists preparing the samples before sequencing, thus
			responding to the needs and taking accout for the technical specificities.
